projects/rawkode.academy/platform/events-service/data-model/integrations/zod.ts

```diff
@@ -0,0 +1,81 @@
+import { eventsTable } from "../schema.ts";
+import { createInsertSchema } from "drizzle-zod";
+import { z } from "zod";
+
+export const EventStatus = z.enum(["scheduled", "cancelled", "rescheduled"]);
+
+export const CreateEvent = createInsertSchema(eventsTable, {
+  id: z.string().min(1).max(255),
+  type: z.enum(["event", "course", "workshop", "live-stream"]),
```

Comment:
For the event service, allowing people to RSVP - does it need to know what type of event it is?